Gemfile

```diff
@@ -6,3 +6,5 @@ group :test do
   gem 'simplecov', require: false, group: :test
   gem 'simplecov-console', require: false, group: :test
 end
+
+gem 'twilio-ruby'
```

Comment:
GENERAL COMMENT ON THE WHOLE THING: 10x better than what I did so take my comments with a pinch of salt. I think some of them are also just where you were still work in progress on implementing the SMS feature. Good job, though :)